CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ You can find its changes [documented below](#070---2021-01-01).
 
 ### Added
 
+- OptionFormatter TextBox Formatter ([#2193] by [@meettitan])
```

Comment:
Unfortunately the links don't get automatically generated. Please define both `[#2193]` and `[@meettitan]` at the bottom of the file.